index.html

```diff
@@ -13,19 +13,19 @@
   </head>
   <body>
     <div class="container">
-      <section class="todo-list-header">
-        <h1>To do list</h1>
+      <section class="c-header">
+        <h1 class="c-header-heading">To do list</h1v b>
       </section>
-      <section class="todo-list-menu">
-        <div class="c-filter-option todo-list-menu__dropdown-filter">
-          <button class="c-filter-option__dropbtn c-filter-option__filter">
+      <section class="c-menu">
+        <div class="c-menu-control c-menu-item--filter">
```

Comment:
There's a few modifiers in here (`--`), I'm not convinced there should be. Is there a good reason to have them in?